meta-cube/recipes-support/overc-utils/source/cube-ctl

```diff
@@ -1246,12 +1247,8 @@ case "${cmd}" in
 	    if [ $? -ne 0 ]; then
 		echo "[INFO] deleting container ${container_name}"
 		cube-cmd ${runc_essential} delete ${container_name} 2> /dev/null
-		if [ -d "${out_dir}/${container_name}" ]; then
-		    echo "[INFO] removing container source .."
-		    rm -rf "${out_dir}/${container_name}"
-		fi
-		machine=$(cube-cfg id)
-		cube-cfg set overc/$machine/oci/${container_name}/:
+		/opt/overc-installer/overc-ctl untrack ${container_name} -f -o ${out_dir}
```

Comment:
This needs to be called from the PATH, or an otherwise relocatable directory. Since any operation using cube-ctl can be via the host installer or on target. It needs to function the same in both environments.